lib/sensors/include/Sensor.h

```diff
@@ -3,10 +3,11 @@
 namespace Sensor {
 class Sensor {
  public:
-  virtual bool read(float &sensorReading)          = 0;
-  virtual bool alternateRead(float &sensorReading) = 0;
-  virtual bool getStatus() const                   = 0;
-  virtual bool reset()                             = 0;
+  virtual float read()                = 0;
+  virtual float alternateRead()       = 0;
+  virtual bool getStatus() const      = 0;
```

Comment:
this was a bit ambiguous, but i decided not to put a nodiscard for the getstatus function. similar reason as to not putting it in the read and alternateread